instagram-app/src/components/CommentSection/CommentContainer.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Comment from './Comment'
+import './CommentContainer.css'
+import CommentInput from './CommentInput';
+import PropTypes from 'prop-types'
+
+class CommentContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
+            comment: ''
+        };
+    }
+
+componentWillReceiveProps(newProps) {
```

Comment:
Note:

These methods are considered legacy and you should avoid them in new code:

UNSAFE_componentWillUpdate()
UNSAFE_componentWillReceiveProps()